tests/helpers/test_dbml.py

```diff
@@ -0,0 +1,628 @@
+import pathlib
+import textwrap
+
+import pytest
+from pydbml import PyDBML  # type: ignore[import-untyped]
+from pydbml.classes import Reference, Table, Column  # type: ignore[import-untyped]
+
+import dlt
+from dlt.common.schema.typing import TColumnSchema, TTableReference, TTableSchema
+from dlt.helpers.dbml import (
+    export_to_dbml,
+    schema_to_dbml,
+    _to_dbml_column,
+    _from_dbml_column,
+    _to_dbml_table,
+    _from_dbml_table,
+    _to_dbml_reference,
+    _from_dbml_reference,
+    _group_tables_by_resource,
+)
+
+
+@pytest.fixture
+def example_schema() -> dlt.Schema:
+    return dlt.Schema.from_dict(
```

Comment:
please take a look at tests/common/cases - there you'll see a few really complex schemas ie `ethereum` one. you can also all the one below to cases